### ISSUE TITLE
zef-deps Red fails

### ISSUE THREAD

**Issue by melezhik** [coke/raku-zef-deps#7]
Full log, please see - https://ci.sparrowhub.io/report/1210 , deps-report

```
Julias-MacBook-Pro:sparrow-plugins alex$  zef-deps Red
# PACKAGE: Red
# PACKAGE: DBIish
# PACKAGE: DB::Pg
# PACKAGE: UUID
# PACKAGE: JSON::Fast
# PACKAGE: Test::META
# PACKAGE: App::RaCoCo
# PACKAGE: NativeHelpers::Blob
# PACKAGE: NativeLibs
# PACKAGE: NativeCall::TypeDiag
# PACKAGE: NativeCall
# PACKAGE: NativeLibs:ver<0.0.7+>:auth<github:salortiz>
# PACKAGE: LibUUID
# PACKAGE: pq:ver<5>:from<native>
No such method 'as' for invocant of type 'Any'
  in block  at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/70DB312A4F16F880D795FCEFFC1630B1FE9359CC (Zef::Repository) line 143
  in method candidates at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/70DB312A4F16F880D795FCEFFC1630B1FE9359CC (Zef::Repository) line 134
  in method find-candidates at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/21E5E178916CFEA5E9F91AC79EE76FC21CB208CD (Zef::Client) line 369
  in method find-candidates at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/21E5E178916CFEA5E9F91AC79EE76FC21CB208CD (Zef::Client) line 350
  in block  at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/69DFCBFD351566D05EF3A77173DAD74DEA124138 (App::Zef-Deps) line 28
  in sub MAIN-handler at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/sources/69DFCBFD351566D05EF3A77173DAD74DEA124138 (App::Zef-Deps) line 18
  in sub MAIN at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/resources/324FD1668E516D479C8A06067AFAEB43CCA0EB10 line 24
  in block <unit> at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/resources/324FD1668E516D479C8A06067AFAEB43CCA0EB10 line 11
  in sub MAIN at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/bin/zef-deps line 3
  in block <unit> at /Users/alex/.rakubrew/versions/moar-2022.06/share/perl6/site/bin/zef-deps line 1
```

**Comment by coke** [comment]
From trying to find the dependencies of a native requirement:

```
% zef info 'pq:ver<5>:from<native>'
Type check failed in assignment to @results; expected Zef::Candidate but got Any (Any)
```

Fixed in 0.9.5; if we get an error like this from the zef modules, we assume no further dependencies.

**Comment by melezhik** [comment]
works now - https://ci.sparrowhub.io/report/1215
